docs/features/async-streams.md

```diff
@@ -117,7 +117,8 @@ But it contains additional state:
 - a promise of a value-or-end,
 - a current yielded value of type `T`,
 - an `int` capturing the id of the thread that created it,
-- a `bool` flag indicating "dispose mode".
+- a `bool` flag indicating "dispose mode",
+- a `CancellationToken` field.
```

Comment:
Consider noting here it's explicitly the value accessible via `cancellationToken`